style.css

```diff
@@ -0,0 +1,84 @@
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 24px;
+  margin-right: 16px;
+  margin-top: 58px;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
```

Comment:
- [ ] Kindly add on the **logo text** the correct `font-size`, `font-weight`, `font-family`, `etc` provided from **Figma** to make your design look smart.

Reply:
Thank you so much.